### ISSUE TITLE
Source for driver or kernel diff?

### ISSUE THREAD

**Issue by brloomis** [UCTRONICS/UCTRONICS_LCD35_RPI#8]
Hi,
Is the source for the driver available, either standalone or as a diff for the kernel source? It's not clear how the kernel images were built.

I see CONFIG_FB_ILI9488=y is set in the config, but that seems to differ from other publicly-available drivers for the ILI9488.

Thanks!

**Comment by BearFather** [comment]
Some sort of patch or source would be nice for those that want to roll their own.  Mainly cause now I have updated my kernel this is now not working.

**Comment by stevester94** [comment]
Agreed, source would be great, especially given that there is basically no documentation....

**Comment by CodyTracy** [comment]
Need source. It's required under the GPL.
